### DISCUSSION TITLE
How do people find available published modules

### DISCUSSION THREAD

**Discussion by strangemonad** [stefanprodan/timoni#388] [Q&A]
I love that modules are just OCI artifacts but I'm curious how people have been finding what timoni modules exist out there in the wild? The common charts repo and search is one of the things helm does quite well.

Maybe if there's a common way to annotated OCI artifacts an index could be made by crawling well-known artifact registries?

**Comment by cgiraldo** [comment]
I am new to timoni, but for helm chart I use https://artifacthub.io/ for search and also publish charts. Maybe It is a good place if they eventually support timoni modules.
